dvc/cli.py

```diff
@@ -144,6 +144,17 @@ def parse_args(argv=None):
         description=desc,
         parents=[parent_parser],
         formatter_class=argparse.RawTextHelpFormatter,
+        add_help=False,
+    )
+
+    # NOTE: adding this argument in get_parent_parser() either in
```

Comment:
Let's clarify in the comment that we are only doing this to make the message start from a capital letter.

Reply:
@Hiyorimi Let's add the comment on why we are doing this. It is not clear from the code. :slightly_smiling_face:

Reply:
Sorry, missed that. Now fixed that.